juju/client/facade.py

```diff
@@ -13,13 +14,22 @@
 from collections import defaultdict
 from glob import glob
 from pathlib import Path
-from typing import Any, Dict, List, Mapping, Sequence
+from typing import Any, Mapping, Sequence
 
 import packaging.version
 import typing_inspect
+from typing_extensions import TypeAlias
 
 from . import codegen
 
+# Plain JSON, what is received from Juju
+_JSON_LEAF: TypeAlias = None | bool | int | float | str
```

Comment:
Does this work in python 3.8, or do you need to use a Union? We have annotations from the future, but these `|`s aren't in an annotation

Reply:
stringified.